layouts/partials/content.html

```diff
@@ -47,6 +48,10 @@ <h2 id="available-scalers">
         <script async src="https://artifacthub.io/artifacthub-widget.js"></script>
       </div>
       {{ end }}
+
+      {{ if $providersListVisible }}
+      {{ partial "providers.html" . }}
+      {{ end }}
```

Comment:
Instead of modifying this `contexnt.html` template, I'd recommend defining a shortcode and then using the shortcode in the relevant pages. Just move `providers.html` to be under shortcodes and then use it like this, e.g., in `authentication.md`:

```

Available authentication providers for KEDA:

{{< providers >}}
```

Reply:
Good point! Done!